docs/cookbooks-retention-expiry.md

```diff
@@ -205,7 +205,7 @@ The following retention policies are available:
 | `consumer_backlog_eviction` | The broker will begin discarding backlog messages                                                 |
 
 > #### Beware the distinction between retention policy types
-> As you may have noticed, there are two definitions of the term "retention policy" in Pulsar, one that applies to persistent storage of messages not in backlogs, and one that applies to messages within backlogs.
+> As you may have noticed, the scope of retention includes the backlog, so retention must be greater than the backlog quota; otherwise, Pulsar will throw an error when set both.
```

Comment:
Fix grammar in this sentence. "when set both" is not correct. "As you may have noticed" could be omitted. Suggestion:
```
:::note
The scope of retention includes the backlog, so retention must be greater than the backlog quota. Otherwise, Pulsar will throw an error when both are set.
:::
```